docs/cephfs-snap-clone.md

```diff
@@ -0,0 +1,79 @@
+# Create CephFS snapshot and Clone Volume
+
+- [Create CephFS snapshot and Clone Volume](#create-cephfs-snapshot-and-clone-volume)
+  - [Prerequisite](#prerequisite)
+  - [Snapshot](#snapshot)
+    - [Create snapshotclass](#create-snapshotclass)
+    - [Create snapshot](#create-snapshot)
+  - [Restore snapshot to a new PVC](#restore-snapshot-to-a-new-pvc)
+  - [Clone PVC](#clone-pvc)
+
+## Prerequisite
+
+- For snapshot functionality to be supported for your Kubernetes cluster, the
+  Kubernetes version running in your cluster should be >= v1.17. We also need the
+  `snapshot controller` deployed in your Kubernetes cluster along with `csi-snapshotter`
+  sidecar container.
+  Refer [external-snapshotter](https://github.com/kubernetes-csi/external-snapshotter/)
+  for more information on these sidecar controllers. There should
+  be a `volumesnapshotclass` object present in the cluster
+  for snapshot request to be satisified.
+
+## Snapshot
+
+### Create snapshotclass
+
+```console
+kubectl create -f snapshotclass.yaml
```

Comment:
Not sure where my comment went... I asked where this file comes from, and if you can explain that in the document.

Reply:
Ah.. I missed it while removing the other commit .. sorry.. bringing it back.

Reply:
@nixpanic ptal .. restored that version :)